functions/src/index.ts

```diff
@@ -232,6 +232,123 @@ function cleanAverages(averages: SensorReading[]): CleanedReadings {
   return new CleanedReadings(latitude, longitude, cleanedAverages);
 }
 
+/**
+ * Computes the AQI for PM 2.5 given the appropriate AQI breakpoints. in most use cases, this function will be called from a function which knows those breakpoints.
+ *
+ * Adapted from EPA function available at https://www.airnow.gov/sites/default/files/custom-js/conc-aqi.js
+ *
+ * @param pm25Concentration - the PM 2.5 concentration to calculate the AQI for
+ * @param lowConcentrationBreakpoint - the low breakpoint for PM 2.5 that the concentration falls within
+ * @param highConcentrationBreakpoint - the high breakpoint for PM 2.5 that the concentration falls within
+ * @param lowIndexBreakpoint - the low breakpoint for AQI that the AQI will fall between
+ * @param highIndexBreakpoint - the high breakpoint for AQI that the AQI will fall between
+ */
+function _indexCalculation(
```

Comment:
Why does this function name start with an underscore?

Reply:
My intention had been to indicate that this was a "private" function, but I think if we are externalizing into the other file as recommended below then that will be unnecessary